cmsplugin_carousel_ai/migrations/0002_auto_20170113_2028.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cmsplugin_carousel_ai', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='slide',
+            name='caption_line1',
+        ),
+        migrations.RemoveField(
+            model_name='slide',
+            name='caption_line2',
+        ),
+        migrations.RemoveField(
+            model_name='slide',
+            name='caption_line3',
+        ),
+        migrations.AddField(
+            model_name='slide',
+            name='slide_content',
+            field=models.CharField(max_length=160, blank=True, verbose_name='slide content'),
+        ),
+        migrations.AddField(
+            model_name='slide',
+            name='slide_header',
+            field=models.CharField(max_length=160, blank=True, verbose_name='slide header'),
+        ),
+        migrations.AddField(
+            model_name='slide',
+            name='slide_subheader',
+            field=models.CharField(max_length=160, blank=True, verbose_name='slide subheader'),
+        ),
+        migrations.AddField(
+            model_name='slide',
+            name='template',
+            field=models.CharField(default='default', max_length=255, verbose_name='Template', choices=[('default', 'Default')]),
```

Comment:
Since the `template` field in `Carousel(CMSPlugin)` defines choices from settings we end up with a migration that has the choices you have locally used. This is not something we want to have in a reusable application.

One solution could be to add the choices just into the form used in the admin and keep the model field free from such restrictions.